client/src/components/ScatterChart.tsx

```diff
@@ -1,22 +1,27 @@
+import { ChartOptions } from 'chart.js';
+import AnnotationPlugin from 'chartjs-plugin-annotation';
 import React from 'react';
-import PropTypes from 'prop-types';
 import { Scatter } from 'react-chartjs-2';
-import AnnotationPlugin from 'chartjs-plugin-annotation';
 
 /**
- * @typedef {object} AxisProps
  * @property {string} label Axis label
  * @property {string} unit  Axis unit of measurement
```

Comment:
This documentation can be moved to be with the variables as in `LogFile` from `files.ts`

Reply:
Done
394dacc